api/app/tag.py

```diff
@@ -196,19 +198,33 @@ async def fetch_metadata(iss: str, product: str):
             code="failed_to_fetch_metadata",
         )
 
+    try:
+        config = await get_dataspace_configuration(product_passport.product_dataspace)
+        gateway = config["product_gateway_url"]
+    except KeyError:
+        logger.exception(
+            f"product_gateway_url not found in {product_passport.product_dataspace} configuration")
+        raise TagsError(
+            error="Product gateway URL not found in dataspace configuration.",
+            code="failed_to_fetch_metadata",
+        )
+
+    pgw_openapi = await fetch_json_file(f"{gateway}/openapi.json")
+
+    sdp_definitions = pgw_openapi.get("paths", {})
```

Comment:
What is `sdp`?

Reply:
"Supported data product"? But this list is not *supported data products*, this is .. data products

Reply:
```suggestion
    definition_paths = pgw_openapi.get("paths", {})
```